recipes/ffbidx/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "1.0.1" %}
+
+{% set cuda_major = (cuda_compiler_version|default("12.0")).rpartition('.')[0] %}
+
+package:
+  name: ffbidx
+  version: {{ version }}
+
+source:
+  url: https://github.com/paulscherrerinstitute/fast-feedback-indexer/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 99fdd8be09862ceba38436ffb1bf34c164d796e6f88f185b3185d4a606020172
+
+build:
+  number: 0
+  skip: True  # [not linux64]
+  skip: True  # [(cuda_compiler_version or "None") == "None"]
+  skip: true  # [cuda_major != "12"]
+
+requirements:
+  host:
+    - {{ compiler('cuda') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - meson
+    - pkg-config
```

Comment:
Usually I would expect these four should be in `build:`. But I don't know if meson needs that to workaround the environment.

Reply:
Ah, meson was reviewed as needing to be in host, above, so maybe it does belong. It probably doesn't matter that much, if it doesn't export a run requirement.

Reply:
It will matter for cross-compilation probably and with having meson here, you avoid having a python installation in build.

Reply:
I think this got mixed up by the GitHub app. Compilers should only be in build.